### ISSUE TITLE
CI error: No such file or directory @ rb_sysopen - /home/travis/build/luma-institute/<app>/log/ember-frontend.test.log

### ISSUE THREAD

**Issue by patcoll** [thoughtbot/ember-cli-rails#332]
I'm getting this error while running controller tests in CI environments. On both Travis CI and CircleCI, in their newest Docker environments.

I'm resorting to the following hack to get around it for now. I don't need to build in production since I run `assets:precompile` before I deploy. My view for my this custom controller uses `render_ember_app` too.

    class AssetsController < ApplicationController
      def build_frontend
        EmberCli[:frontend].build
      end
    
      def run_build?
        Rails.env.development? || (Rails.env.test? && ENV['CI'].blank?)
      end
    
      def index
        build_frontend if run_build?
      end
    end

**Comment by patcoll** [comment]
I had removed the `log/.keep` file from my git repo that Rails installs by default, which seemed to be preventing `ember-cli-rails` from creating the appropriate log files. Apparently some code somewhere assumes the `log` folder exists. After I added this file back, tests started passing in CI again, on both Travis CI and CircleCI.

Silly error on my part, but should `ember-cli-rails` compensate for human error in this case? Asking genuinely. How assumed is it in the Rails community that the log folder will exist?
